brave/brave-core
# Sidebar: Remove feature flag, enable by default, and deprecate "Show on click" option

Fix https://github.com/brave/brave-browser/issues/20876

With this change, sidebar will be available on all platform.
and sidebar show option is "Always" by default.

Security review - https://github.com/brave/security/issues/746

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
`npm run test brave_unit_tests -- --filter=SidebarService*`

1. Launch browser and check sidebar is not shown
2. Check sidebar entry is visible from app menu and it has three options - show/hover/never

* Sidebar `On click` show option deprecation test
1. Launch prev brave version and set show option as On click and close.
2. Launch latest brave version and check show option is set as `On mouse over`
